### PR TITLE
#162235446 Setup babel to transpile to ES5

### PR DESCRIPTION
**Pull request by akhilome**
#### What does this PR do?

Sets up babel to transpile ES6 to ES5

#### Description of Task to be completed?

- install babel-cli, required presets, and plugins
- setup build script and ignore distribution directory

#### How should this be manually tested?

N/A

#### Any background context you want to provide?

N/A

#### What are the relevant pivotal tracker stories?

[#162235446](https://www.pivotaltracker.com/story/show/162235446)

#### Screenshots (if appropriate)

N/A

#### Questions:

N/A

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "presets": ["env"],
+  "plugins": ["transform-object-rest-spread"]
+ }
+
```
Comment (omokaro-faith) [inline .babelrc]: A line break is needed here.

**Comment by omokaro-faith** [comment]
@akhilome Fix Conflicts

File: .babelrc (hunk 1 above)
Comment (akhilome) [reply .babelrc]: Thanks for pointing that out, fixed now.

**Comment by akhilome** [comment]
@omokaro-faith all requested changes have been implemented.

**Pull request merged**